solutions/usaco-762.mdx

```diff
@@ -0,0 +1,53 @@
+---
+id: usaco-762
+source: USACO Silver 2017 December
+title: My Cow Ate My Homework
+author: Juheon Rhee
+---
+
+[Official Analysis (C++ and Java)](http://www.usaco.org/current/data/sol_homework_silver_dec17.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$
+
+<LanguageSection>
+
+<PySection>
+
+```python
+with open('homework.in') as read:
+        n = int(read.readline())
+        questions = [int(i) for i in read.readline().split()]
+
+left, right = -1, n
+        
+total = [0] * n
+minimum = [0] * n
+max_score_values = []
+
+total[n - 1] = questions[n - 1]
```

Comment:
You might as well initialize `total = list(questions)` (and similarly for `minimum`.

Reply:
yeah but I only need the value in questions[n-1]